### PR TITLE
feature: modify defaut bridge mode

### PR DESCRIPTION
**Pull request by rudyfly**
### Ⅰ. Describe what this PR did
Change the default nat bridge network from 172.17.0.1/24 to 192.168.5.1/24
Complete the bridge network configure.
Don't to change bridge device ip address.

### Ⅴ. Special notes for reviews

Signed-off-by: Rudy Zhang <rudyflyzhang@gmail.com>

### CODE REVIEW

File: daemon/config/config.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ func (cfg *Config) Validate() error {
 }
 
 //MergeConfigurations merges flagSet flags and config file flags into Config.
```
Comment (allencloud) [inline daemon/config/config.go]: Please update this comment as well.

File: extra/libnetwork/drivers/bridge/setup_ipv4.go [hunk 2]
```diff
@@ -16,13 +17,13 @@ func setupBridgeIPv4(config *networkConfiguration, i *bridgeInterface) error {
 		return fmt.Errorf("failed to retrieve bridge interface addresses: %v", err)
 	}
 
-	if !types.CompareIPNet(addrv4.IPNet, config.AddressIPv4) {
+	if os.Getenv("SetBridgeIP") == "true" && !types.CompareIPNet(addrv4.IPNet, config.AddressIPv4) {
```
Comment (allencloud) [inline extra/libnetwork/drivers/bridge/setup_ipv4.go]: This change should add something into the document, right?
For this kind of work, the pouchd's source code relies on some outter specific environment variables. If no guidance is here, it is quite hard to software user to notice the usage of env. WDYT?
How to solve the problem for user is what we need to take into consideration. @rudyfly

File: main.go [hunk 3]
```diff
@@ -99,6 +99,16 @@ func setupFlags(cmd *cobra.Command) {
 	// volume config
 	flagSet.StringVar(&cfg.VolumeConfig.DriverAlias, "volume-driver-alias", "", "Set volume driver alias, <name=alias>[;name1=alias1]")
 
+	// network config
+	flagSet.StringVar(&cfg.NetworkConfig.BridgeConfig.Name, "bridge-name", "", "Set default bridge name")
+	flagSet.StringVar(&cfg.NetworkConfig.BridgeConfig.IP, "bip", "", "Set bridge ip")
+	flagSet.StringVar(&cfg.NetworkConfig.BridgeConfig.GatewayIPv4, "default-gateway", "", "Set default bridge gateway")
```
Comment (allencloud) [inline main.go]: Set IP address for default bridge gateway?

In addition, when we use IP in description or document, we must use IP with both letters uppercased. Please update the block of code. 

MTU and CIDR as well.

File: network/config.go [hunk 4]
```diff
@@ -5,33 +5,32 @@ var DefaultExecRoot = "/var/run/pouch"
...
-	Name        string
-	IP          string
-	FixedCIDR   string
-	GatewayIPv4 string
-	PreferredIP string
+	Name        string `json:"bridge-name"`
+	IP          string `json:"bip"`
+	FixedCIDR   string `json:"fixed-cidr"`
+	GatewayIPv4 string `json:"default-gateway"`
+	PreferredIP string `json:"preferred-ip"`
```
Comment (allencloud) [inline network/config.go]: I think it is quite hard to understand this field without annotation.

File: main.go (hunk 3 above)
Comment (rudyfly) [reply main.go]: > Set IP address for default bridge gateway?
yes, it is bridge config, so set bridge ip address

> In addition, when we use IP in description or document, we must use IP with both letters uppercased. Please update the block of code.
>
> MTU and CIDR as well.
OK, I will modify

File: extra/libnetwork/drivers/bridge/setup_ipv4.go (hunk 2 above)
Comment (rudyfly) [reply extra/libnetwork/drivers/bridge/setup_ipv4.go]: I will change it in my next job that vendor libnetwork into alibaba group.

File: daemon/config/config.go (hunk 1 above)
Comment (rudyfly) [reply daemon/config/config.go]: I think this comments is right 😄

**Comment by allencloud** [comment]
LGTM

**Pull request merged**